src/ui/components/ExperimentBrowserPage.tsx

```diff
@@ -103,7 +103,7 @@ const ExperimentBrowserPage: FC = () => {
           addToast({
             message: `Id successfully generated and copied to clipboard. Test Id: ${result}`,
             variant: "success",
-            autohide: true,
+            autohide: false,
```

Comment:
why is this change here?

Reply:
When I renamed the parameter from `permanent ` to `autohide`, I forgot to flip the boolean values since they are kind of opposites so I just made that change here.